qemu/tests/cfg/virtio_mem_numa_basic.cfg

```diff
@@ -40,3 +40,10 @@
     variants operation:
         - @default:
         - with_reboot:
+        - with_migration:
+            max_vms = 2
+            migration_test_command = help
```

Comment:
It looks like this cmd is not called, the other part is ok for me :)

Reply:
Changed now, thanks!